_posts/en/newsletters/2022-02-16-newsletter.md

```diff
@@ -57,10 +57,10 @@ wallets and services.*
   performance] in the [1.6.0 release][fulcrum 1.6.0].
 
 - **Kraken announces proof of reserves scheme:**
-  [Kraken details][kraken por] their proof of reserves scheme involving a
+  Kraken [details][kraken por] their proof of reserves scheme involving a
   trusted auditor, also noting shortcomings and future improvements. Kraken
-  proves onchain address ownership by digitally signing,
-  a merkle tree of Kraken user account balances is created, the auditor verifies
+  proves onchain address ownership by digitally signing
```

Comment:
@harding Im not sure this is right. I believe Kraken is signing to prove ownership of UTXOs onchain. Not signing a merkle tree

Reply:
Sorry!  Hopefully fixed in the latest push.

Reply:
That mention of Kraken's POR made me wonder how they show each user separate UTXOs, and especially how they prove that. It doesn't seem particularly scalable to maintain each user's balance as distinct UTXOs.

Reply:
Ah nevermind. Rereading that section, I realize now that the user balances on the exchange was included in the total and the notary certifies that the on-chain balance exceeds all deposits. It's much clearer now!